src/AWS4AuthRequest.jl

```diff
@@ -111,6 +111,11 @@ function sign_aws4!(method::String,
         if k == "x-amz-security-token" && !token_in_signature
             continue
         end
+        # In Amazon's examples, they exclude Content-Length from signing. This does not
+        # appear to be addressed in the documentation, so we'll just mimic the example.
+        if k == "content-length"
+            continue
+        end
```

Comment:
I'd make this a one-liner but that's just me ¯\_(ツ)_/¯

Reply:
I was going to, it just looked a bit funny to me with the long-form `if` above it. I can change it if that's better, I don't feel strongly (or have any particular feelings at all) about it.

Reply:
Yeah, going back and changing it now I think it looks out of place since there are long-form `if`s both above and below it. So I'll opt to keep it this time.